src/docs/product/accounts/pricing.mdx

```diff
@@ -40,9 +40,21 @@ The second is the **On-Demand Period**. The On-Demand Period reflects a one mont
 
 ### Billing Controls
 
-#### On-Demand Spending Cap {#on-demand-cap}
+#### On-Demand Budgets {#on-demand-cap}
 
-You can set a maximum monthly on-demand bill amount by setting an on-demand spending cap. Once you reach the spending cap, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+You can set a maximum monthly on-demand invoice amount by setting an on-demand budget. Once you reach the maximum budget capacity, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+
+<Note>
+
+Per-category on-demand budgets are only available on plans with [performance monitoring](/product/performance/) features. Customers on legacy plans can only use the shared on-demand budget strategy. 
```

Comment:
We should probably avoid the use of the word "legacy" just we don't open a discussion of "what does legacy" mean. 

```suggestion
Per-category on-demand budgets are only available on plans with [performance monitoring](/product/performance/) features. Customers on plans without performance monitoring can only use the shared on-demand budget strategy. 
```